src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -147,6 +157,41 @@ export default class PatientViewMutationTable extends MutationTable<
                 PanelColumnFormatter.getGenePanelIds(GenePanelProps(d)),
         };
 
+        // customization for ASCN-related columns
+        // Patient view differs from default/results view because multiple samples can appear in a row
+        // due to same variant in multiple samples
+        // This can lead to cases where there are multiple icons/tooltips in a single cell
+        // therefore patient view needs sampleManager to indicate which values match which samples
+
+        this._columns[
+            MutationTableColumnType.CANCER_CELL_FRACTION
+        ] = getDefaultCancerCellFractionColumnDefinition(
+            this.getSamples(),
+            this.props.sampleManager ? this.props.sampleManager : undefined
```

Comment:
i think you can just pass this.props.sampleManager as this condition will need to be handled inside the function anyway.

Reply:
@sheridancbio double check

Reply:
Here is my take on this ... I think @averyniceday is concerned that maybe accessing this.props.sampleManager when it is not defined may generate some kind of runtime error. I think we do not need to be concerned about that because other properties are used within this function (generateColumns()), such as on line 126/136, where the sampleManager is passed as part of a lambda function for the sortBy member during construction. If it does not generate a runtime error there, it should not generate an error here.

Reply:
I traced this into CancerCellFractionColumnFormatter.renderFunction and then into CancerCellFractionElement.tsx. It appears to me that the argument passed into these functions is not actually used ... in CancerCellFractionElement, it seems to refer back to the global "props" in order to access sampleManager. So I think there may be no issue here. Perhaps in the future we can refactor this code to remove the argument / data member from these classes, if the global "props" is accessible down at the level where the component is being rendered ... but I am not certain of this.

Reply:
Tracing again with @averyniceday, I see that the argument holding the sampleManager actually is used .. when rendering the tooltip for the CancerCellFractionElement.
In that code, there is a check whether the sampleManager is set or not (using the trinary operator in javascript). The javascript trinary operator treats a value of 'undefined' the same as a 'null' value (or NaN or 0) ... these all select the ifFalse branch of the expression. So it appears that the usage down at the element level already *is* properly handling the case when no sampleManager is available.

Reply:
I would like to keep this as is -- this was introduced following a template provided by Hongxin. Other columns below are following a similar pattern. We would like to avoid making adjustments that do not touch the business logic if possible.